src/interfaces/IStakeRegistry.sol

```diff
@@ -231,8 +236,15 @@ interface IStakeRegistry is IRegistry {
     function getCurrentTotalStakeForQuorum(uint8 quorumNumber) external view returns (uint96);
 
     /**
-     * @notice Used for updating information on deposits of nodes.
-     * @param operators are the addresses of the operators whose stake information is getting updated
+     * @notice Called by the registry coordinator to update an operator's stake for one
+     * or more quorums.
+     *
```

Comment:
Add @return natspec here. We should probably be more consistent on our natspec docs throughout but especially for interfaces where we don't see the implementation.

Reply:
... and [35247f8](https://github.com/Layr-Labs/eigenlayer-middleware/pull/57/commits/35247f8e1458ae2a50f32192141c2ec76f204ec1) because me dumb